src/components/info/DeleteModal.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+	import Modal from "../Modal.svelte";
+	import { createEventDispatcher } from 'svelte';
+
+	const dispatch = createEventDispatcher();
+
+	export let show: boolean = false;
+
+	let confirm = "";
+	$: confirmValid = confirm.toLowerCase().includes("delete this entry");
+
```

Comment:
I think it should probably be an exact match -- I can't help but imagine someone writing "do not delete this entry!"

Reply:
ah, true. I was trying to account for the case of someone writing `"delete this entry"` including the quotation marks, but I can see how that might be a problem.